src/main/java/meditracker/argument/ArgumentName.java

```diff
@@ -19,7 +19,10 @@ public enum ArgumentName {
     HELP("help"),
     MORNING("morning"),
     AFTERNOON("afternoon"),
-    EVENING("evening");
+    EVENING("evening"),
+    ILLNESS("illness"),
+    SIDE_EFFECTS("sideEffects"),
+    ALL_FIELDS("findAllFields");
```

Comment:
Please follow the naming convention as per described in the DG
```suggestion
    ALL_FIELDS("allFields");
```